app/src/main/java/com/andrew/tasky/presentation/agenda/AgendaViewModel.kt

```diff
@@ -54,25 +50,25 @@ class AgendaViewModel@Inject constructor(
     }
 
     init {
-        val daysBefore = 6
-        val daysAfter = 8
 
-        for (day in 1..daysBefore) {
-            _calendarDateItemList.value += CalendarDateItem(
-                false,
-                LocalDate.now().minusDays(day.toLong())
+        val daysAfterCurrentDate = 5
+        var calendarList = listOf<CalendarDateItem>(
+            CalendarDateItem(
+                true,
+                LocalDate.now()
             )
```

Comment:
See, here it's now unclear for me what this `true` stands for. Could literally be anything 😁